_inc/client/apps/style.scss

```diff
@@ -0,0 +1,218 @@
+.jp-landing__apps.dops-card {
+	padding: 0;
+
+	.dops-button {
```

Comment:
I can certainly tighten up a lot of the styles still. I didn't completely rewrite everything, but now that you made a few comments, this code is irking me as well.